src/AppBundle/Manager/SubscriptionManager.php

```diff
@@ -93,14 +120,31 @@ public function saveSubscription(Subscription $subscription)
     {
         $this->em->persist($subscription);
         $this->em->flush($subscription);
+
+        $this->dispatcher->dispatch(
+            SubscriptionEvents::POST_WRITE,
+            new SubscriptionEvent($subscription->getId(), $subscription)
+        );
     }
 
     /**
      * @param Subscription $subscription
+     * @param bool $dispatch
      */
-    public function updateSubscription(Subscription $subscription)
-    {
+    public function updateSubscription(
+        Subscription $subscription,
+        $dispatch = true
+    ) {
         $this->em->flush($subscription);
+
+        if (!$dispatch) {
+            return;
+        }
+
+        $this->dispatcher->dispatch(
```

Comment:
Als er een param voor lezen is zal er ook 1 voor schrijven moeten zijn, anders kom je ook hier in een oneindige loop.